dubbo-monitor/dubbo-monitor-default/src/main/java/org/apache/dubbo/monitor/dubbo/CpuUsageServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.dubbo.monitor.dubbo;
```

Comment:
I'm not sure if should be at package org.apache.dubbo.monitor.dubbo; or move to dubbo-cluster

Reply:
Hi, I think put it under dubbo-cluster is ok.  Maybe you can create a new package called `org.apache.dubbo.rpc.cluster.loadbalance.statistics`, and put it there.